bindata/bootkube/manifests/configmap-admin-kubeconfig-client-ca.yaml

```diff
@@ -3,6 +3,8 @@ kind: ConfigMap
 metadata:
   name: admin-kubeconfig-client-ca
   namespace: openshift-config
```

Comment:
things in this namespace cannot be replaced.

Reply:
sure, but we can still report the hours till these expire via prometheus.